default-config.json

```diff
@@ -50,20 +50,30 @@
       "alias": "Cadence play "
     }
   ],
+  "commandFriendlyNames": {
+    "play": "play",
```

Comment:
Just a thought - Is it worse to pollute config with these friendly names that match the key, or... Maybe we could default the friendly name to the key if it's not present?

Hm, that seems harder than it's worth. But just a thought...

Reply:
Hmm, I think it'll be easy enough on the code side and intuitive enough from the config side so I'll add it then unless you decide against it

Reply:
If you think a default isnt too much trouble I'm a fan. Just don't invest too much effort into it, I'm not demanding you add it.